openedx/core/djangoapps/content_libraries/views.py

```diff
@@ -582,6 +614,27 @@ def delete(self, request, usage_key_str):  # pylint: disable=unused-argument
         return Response({})
 
 
+@view_auth_classes()
+class LibraryBlockLtiUrlView(APIView):
+    """
+    Views to generate LTI URL for existing XBlocks in a content library.
+
+    Returns 404 in case the block not found by the given key.
+    """
+    @convert_exceptions
+    def get(self, request, usage_key_str):
+        """
+        Get the LTI login URL for the XBlock.
```

Comment:
This is should actually be the "LTI launch URL."

Reply:
👍 I typoed it. Will fix today/tomorrow.